docs/docs/reference/policies.mdx

```diff
@@ -180,7 +180,9 @@ For example
   ```
 
 ## Policy scripts format (REGO)
```

Comment:
I'd personally add a section here called "how to write a Chainloop policy" and just add a link to the guide